isis/src/control/objs/BundleAdjust/BundleAdjust.cpp

```diff
@@ -1896,237 +1880,65 @@ namespace Isis {
-                                             &coeffPoint3D(0, 1),
-                                             &coeffPoint3D(1, 1));
-    measureCamera->GroundMap()->GetdXYdPoint(lookBWRTCoord3,
-                                             &coeffPoint3D(0, 2),
-                                             &coeffPoint3D(1, 2));
+    // Retrieve the coordinate type (latitudinal or rectangular) and compute the partials for
+    // the fixed point with respect to each coordinate in Body-Fixed
+    SurfacePoint::CoordinateType coordType = m_bundleSettings->controlPointCoordTypeBundle();
+    observation->computePoint3DPartials(coeffPoint3D, measure, coordType);
 
     // right-hand side (measured - computed)
-    measuredX = measure.focalPlaneMeasuredX();
-    measuredY = measure.focalPlaneMeasuredY();
-
-    deltaX = measuredX - computedX;
-    deltaY = measuredY - computedY;
-
-    coeffRHS(0) = deltaX;
-    coeffRHS(1) = deltaY;
+    observation->computeRHSPartials(coeffRHS, measure);
 
-    // residual prob distribution is calculated even if there is no maximum likelihood estimation
-    double obsValue = deltaX / measureCamera->PixelPitch();
-    m_bundleResults.addResidualsProbabilityDistributionObservation(obsValue);
+    double deltaX = coeffRHS(0);
+    double deltaY = coeffRHS(1);
 
-    obsValue = deltaY / measureCamera->PixelPitch();
-    m_bundleResults.addResidualsProbabilityDistributionObservation(obsValue);
-
-    observationSigma = 1.4 * measureCamera->PixelPitch();
-    observationWeight = 1.0 / observationSigma;
+    m_bundleResults.addResidualsProbabilityDistributionObservation(observation->computeObservationValue(measure, deltaX));
+    m_bundleResults.addResidualsProbabilityDistributionObservation(observation->computeObservationValue(measure, deltaY));
 
+    double observationWeight = 1.0; 
     if (m_bundleResults.numberMaximumLikelihoodModels()
           > m_bundleResults.maximumLikelihoodModelIndex()) {
-      // if maximum likelihood estimation is being used
+      // If maximum likelihood estimation is being used
       double residualR2ZScore
-                 = sqrt(deltaX * deltaX + deltaY * deltaY) / observationSigma / sqrt(2.0);
-      //dynamically build the cumulative probability distribution of the R^2 residual Z Scores
+                 = sqrt(deltaX * deltaX + deltaY * deltaY) / sqrt(2.0);
+
+      // Dynamically build the cumulative probability distribution of the R^2 residual Z Scores
       m_bundleResults.addProbabilityDistributionObservation(residualR2ZScore);
+
       int currentModelIndex = m_bundleResults.maximumLikelihoodModelIndex();
       observationWeight *= m_bundleResults.maximumLikelihoodModelWFunc(currentModelIndex)
                             .sqrtWeightScaler(residualR2ZScore);
     }
 
-    // multiply coefficients by observation weight
     coeffImage *= observationWeight;
     coeffPoint3D *= observationWeight;
     coeffRHS *= observationWeight;
```

Comment:
These needed to stay in in case the maximum likelihood is used and needs to update the weighting. The default weighting has now been moved into the observation classes.

Reply:
Could this just be moved into the if using maximum likelihood block? That would better isolate it.

Reply:
Good point! Not sure why I didn't do that in the first place!